CHANGELOG/README.md

```diff
@@ -0,0 +1,98 @@
+# CHANGELOG
+
+The CHANGELOG directory is used for individual changelog file practice.
+The `kong/CHANGELOG.md` now is deprecated.
+
+
+## How to add a changelog file for your PR?
+
```

Comment:
Let's automate the steps below.  `changelog add` should create the file from the template, possibly asking the user for the PR number.  The name of the file that was created should be printed, and if the `EDITOR` environment variable is set, the editor should be started on that file.

Reply:
The `changelog add` command is designed to be used by GitHub workflow. Preferring keep it simple. The contributors shouldn't be distracted by adding the file. They only edit it if they are unsatisfied with the auto-generate. Before the auto CI comes up, I prefer to manually copy and edit.

Reply:
This is the first time I hear about a "GitHub" workflow.  It would help if you could provide some documentation that describes how this tool is supposed to work and be used.